backend/handler/class.go

```diff
@@ -83,3 +83,50 @@ func CreateInvite(cc echo.Context) error {
 		"inviteCode": inviteCode,
 	})
 }
+
+// Functionality : add User to a class Endpoint
+// step1 : verify if classId exists
+// step2 : get the user type
+// step3 : add user to ClassMembers table
+// No body content required as of now
+func EnrollStudent(cc echo.Context) error {
+	c := cc.(*Context)
+	classId := c.Get("classId")
+
+	// verify classId
+	if err := c.Conn.QueryRowContext(c, `
+	SELECT id
+	FROM Classes
+	WHERE id = $1
+	`, classId).Scan(&classId); err != nil {
+		return c.NoContent(http.StatusInternalServerError)
+	}
```

Comment:
This part makes sense, but since our `ClassMembers` table enforces the foreign key constraint that `class_id` exists in the `Classes` table, we don't need to check this, actually.